line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingClient.java

```diff
@@ -337,6 +338,13 @@
      */
     CompletableFuture<GetNumberOfFollowersResponse> getNumberOfFollowersResponse(String date);
 
+    /**
+     * Retrieves the demographic attributes for a bot's friends.
+     *
+     * @see <a href="https://developers.line.biz/en/reference/messaging-api/#get-demographic">Get friends demographics</a>
+     */
+    CompletableFuture<GetFriendsDemographicsResponse> getFriendsDemographicsResponse();
```

Comment:
```suggestion
    CompletableFuture<GetFriendsDemographicsResponse> getFriendsDemographics();
```

`Response` is not needed for method signature.